src/collections/blog/2022-01-07-Recap-KubeCon-China-TAG-Network-talk/index.mdx

```diff
@@ -0,0 +1,120 @@
+</p>
+<p>
+    As an introduction to TAG Network, Owens  describes  the group’s  core objectives and reasons for its formation, summarized below:
+</p>
+<ul>
+<li>Listening to ideas and collaborations</li>
+<li>Assisting and attracting projects</li>
+<li>Building communities and providing impartial stewardship</li>
+</ul>
+<p>
+    Calcote and Owens are frequently asked a common question <em>“Is it important for the organization to be a part of CNCF to participate in the TAG Network group?”</em>. Owens emphasizes that anyone can participate, regardless of whether or not their company is a CNCF member organization. It does not require any payment and is an open-source effort in its spirit.
+</p>
+<h3>CNCF TAG Network Projects and various stages they go through:</h3>
+<img src={Projects} className="slides-right" align="center" alt="Tag Networks Projects" />
+
+<p>
+    One of the important goals of the TAG Network is to identify the unique projects that want to participate and shepherd their journey through CNCF project levels. Project-fit and the process of assessment for inclusion in the CNCF require your project to be on terms with the following essential traits:
+</p>
+<ul>
+<li>Uniqueness</li>
+<li>Problem it solves</li>
+<li>Value it brings to the community</li> 
+<li>Market demand</li>
+<li>Diversification </li>
+</ul>
+<p>
+    Creating and stewarding any one of these projects is a significant undertaking. The  project's maintainers pitch the project and explicate how their project meets the above traits. 
+</p>
+<p>
+    You start in a sandbox and work your way up to an incubation area once you've demonstrated a fundamental interest and understanding of the space, as well as some adoption. To graduate, the project must satisfy the required traits, be open-source, provide value and solve a problem effectively. We don't want to promote a CNCF project that is solely driven by one source. Diversification is crucial to the core of any CNCF project. Owens shared a subtle analogy, <em>"a truly open source project is one in which contributors belong to the Companies A+B+C...Z instead of just company A+A+A..."</em> 
+</p>
+
+
+<h4>Working Groups and papers, and how they work.</h4>
+<ul>
+<li>Universal Data Plane API and Service Mesh WG are the two working groups stewarded by TAG Network.</li>
+<li>They utilize Kubecon to educate about the functioning of working groups.</li>
+<li>Whitepapers and publications are carried out by working group members. A Recent publication was published in the IEEE Bridge magazine, entitled <a to="https://www.nxtbook.com/nxtbooks/ieee/bridge_issue3_2021/index.php#/p/16">Analyzing Service Mesh Performance</a>.</li>
```

Comment:
This should link to our resource article published on our own site. Always seek internal links first.